gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageWriteChannel.java

```diff
@@ -83,8 +83,8 @@ public long size() throws IOException {
 
   @Override
   public SeekableByteChannel truncate(long newSize) throws IOException {
-    // TODO(b/18997913): Emulate this functionality by closing and rewriting old file up to newSize.
-    //                   Or maybe just swap out GcsService for the Apiary client.
+    // TODO: Emulate this functionality by closing and rewriting old file up to newSize.
+    //       Or maybe just swap out GcsStorage for the API client.
     throw new UnsupportedOperationException();
   }
 
```

Comment:
My understanding is that you're commenting on your own code here, and not asking me to do anything?